assets/js/util.ts

```diff
@@ -46,6 +46,65 @@ export const abbreviation = (line: string) => {
   }
 };
 
+export const translateRouteID = (id: string) => {
+  switch (id) {
+    case "red":
+      return "Red Line";
+    case "mattapan":
+      return "Mattapan";
+    case "orange":
+      return "Orange Line";
+    case "blue":
+      return "Blue Line";
+    case "silver":
+      return "Silver Line";
+    case "green":
+    case "green-b":
+    case "green-c":
+    case "green-d":
+    case "green-e":
```

Comment:
hmm I'm not sure how much of this is specified in the spec, but would it be worth including strings for the different green branches? I think it's useful to distinguish when an alert is affecting just one branch on not other on the green line trunk, for example